README.md

```diff
@@ -1,39 +1,61 @@
-# Tyk-K8S-bootstrap
+# Tyk-K8S-Bootstrap
 
-This is a standalone app meant to help with bootstrap and deletion of the tyk-stack when installed
-via tyk-helm-charts.
+This is a standalone app meant to help with bootstrap and deletion of the [tyk-stack](https://github.com/TykTechnologies/tyk-charts/tree/main/tyk-stack) when installed
+via Helm Charts
 
-**Note:** 
-<br>This app is needed only for Tyk [Self-managed](https://tyk.io/docs/tyk-on-premises/) deployment!
-<br>[Tyk OSS](https://tyk.io/docs/apim/open-source/) doesn't have a special bootstrap and in [Tyk Cloud](https://tyk.io/docs/tyk-cloud/) it is done for you (being a SaaS).
+> [!NOTE]
+This app is needed only for [Tyk Self-Managed](https://tyk.io/docs/tyk-on-premises/) deployment!
```

Comment:
It's not true anymore. Since we can use bootstrap app to create operator secret.

Reply:
updated 👍